pkg/scheduler/scheduler.go

```diff
@@ -231,6 +209,40 @@ func New(client clientset.Interface,
 	return sched, nil
 }
 
+func InitPolicyFromFile(policyFile string, policy *schedulerapi.Policy) error {
```

Comment:
Do we need this to be exported? It was not previously so there are apparently no users for this (yet).

If it's exported it should have a comment documenting it.

Reply:
I think we should export it for clean code and reduce its complexity.

Reply:
Thanks for your advice, it should be added annotation actually.

Reply:
+1 to what @miguelbernadi said. These functions shouldn't be exported.

Reply:
I just think about the point of clean code. Function should be small and do one thing, so I think we can extract the method for clean code and reduce its complexity.

Reply:
Sorry it seems we miscommunicated. I understand your point on extracting the code into a function. My concern is about exporting it outside the package (Upper-case methods are public in the package, therefore exported).

It would be fine just to make the methods package private, starting with lowercase, like:
```func initPolicyFromFile(...```

Reply:
oh, misunderstand really, I have modified, thanks for your advice.